app/Http/Controllers/Api/Administration/MeTimezoneController.php

```
@@ -14,15 +15,15 @@ class MeTimezoneController extends Controller
     /**
      * Get the current user's timezone.
      */
-    public function show(Request $request)
+    public function show(): TimezoneResource
     {
-        return response()->json(['timezone' => Auth::user()->timezone]);
+        return new TimezoneResource(Auth::user());
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Auth' in method 'show'. (Clean Code Rules)

(StaticAccess)